README.md

```diff
@@ -13,14 +13,14 @@ Also, don't forget to check out [the documentation](https://aka.ms/dotnet-cli-do
 
 Changes in issue triaging
 -------------------------
-As it was explained in the ["Changes to project.json" blog post](https://blogs.msdn.microsoft.com/dotnet/2016/05/23/changes-to-project-json/), we are making significant changes to the CLI tooling by supporting MSBuild as the build engine and `csproj` as the project format. As we are starting that work, we've realized that this also means that many of the shortcomings of the current tooling will be solved by this move. Given this, we've decided to introduce two new labels for issues to indicate that. These labels are: 
+As it was explained in the ["Changes to project.json" blog post](https://blogs.msdn.microsoft.com/dotnet/2016/05/23/changes-to-project-json/), we are making significant changes to the CLI tooling by supporting MSBuild as the build engine and `csproj` as the project format. We've realized that this also means that many of the shortcomings of the current tooling will be solved by this move. Given this, we've decided to introduce two new labels for issues to indicate that. These labels are: 
```

Comment:
There was two things I was wanting to resolve in my earlier feedback:

- "As it was explained" - Sounds like it comes from a very old book.
- The sentence is passive voice. Try to always use active. It's more pleasant to read.